src/Components/ContentSelection/ContentSelection.js

```diff
@@ -25,13 +37,14 @@ const ContentSelection = () => {
           <button
             className={style.button}
             aria-label="somatic exercise"
-            name="somatic exercise"
+            name="SOMATIC"
             onClick={fetchVideo}
           >
             <img
               className={style.somaticIcon}
               src={somaticContentIcon}
               alt="human head with arrows connecting brain to body"
```

Comment:
Sweet alt tags. And I don't think I said it but @Abdeboskey the icons you drew are AH-mazing 🤩